avocado/core/parser.py

```diff
@@ -125,6 +125,9 @@ def start(self):
             dest='subcommand')
 
         # Allow overriding default params by plugins
+        self.args.mux = multiplexer.Mux()
+        # FIXME: Backward compatibility params, to be removed when 36 LTS is
+        # discontinued
```

Comment:
I actually thought about selftest with special keywords like `COMPATIBILITY_36LTS` where the selftest would check for existence of such keywords (the selftest would be updated with the list of banned words). But I didn't wanted to spend too much time on it. Perhaps I should create a trello card for it.